great_tables/_utils.py

```diff
@@ -1,9 +1,24 @@
 from typing import Optional, Union, List, Any
 import pandas as pd
+import importlib
+from types import ModuleType
 import json
 import re
 
 
+def _try_import(name: str, pip_install_line: Optional[str] = None) -> ModuleType:
+    try:
+        return importlib.import_module(name)
+    except ImportError:
+        if pip_install_line is not None:
+            raise ImportError(
+                f"""Module {name} not found
+                - run `{pip_install_line}` to install."""
+            )
```

Comment:
```suggestion
            raise ImportError(
                f"Module {name} not found. Run the following to install."
                f"\n\n`{pip_install_line}`"
            ) from None
```

The spacing of this multiline string deeply indents the `- run...` part.

<img width="945" alt="image" src="https://github.com/posit-dev/great-tables/assets/2574498/4c1ca85d-04d6-43f2-a254-30000eecca2b">

I added a `from None`, so the stack trace isn't so deep!